client/src/templates/Introduction/components/challenges.tsx

```diff
@@ -0,0 +1,124 @@
+import { Link } from 'gatsby';
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import type { Dispatch } from 'redux';
+
+import GreenNotCompleted from '../../../assets/icons/green-not-completed';
+import GreenPass from '../../../assets/icons/green-pass';
+import { executeGA } from '../../../redux';
+import { SuperBlocks } from '../../../../../config/certification-settings';
+import { ExecuteGaArg } from '../../../pages/donate';
+import { ChallengeWithCompletedNode } from '../../../redux/prop-types';
+
+const mapDispatchToProps = (dispatch: Dispatch) =>
+  bindActionCreators({ executeGA }, dispatch);
+
+interface Challenges {
+  challengesWithCompleted: ChallengeWithCompletedNode[];
+  executeGA: (payload: ExecuteGaArg) => void;
+  isProjectBlock: boolean;
+  superBlock: string;
```

Comment:
This will need the import

```suggestion
  superBlock: SuperBlocks;
```

Reply:
Looks like it has the import